code/modules/cargo/packs/security.dm

```diff
@@ -357,3 +357,26 @@
 	cost = CARGO_CRATE_VALUE * 7
 	contains = list(/obj/item/storage/belt/holster/thermal = 2)
 	crate_name = "thermal pistol crate"
+	
+/datum/supply_pack/security/armory/wt550
+	name = "Recalled Weapon Pack"
+	desc = "Contains a set of old Nanotrasen brand autorifles recalled due to choking hazard."
+	cost = CARGO_CRATE_VALUE * 7
+	hidden = TRUE
+	contains = list(
+		/obj/item/gun/ballistic/automatic/wt550 = 2,
+		/obj/item/ammo_box/magazine/wt550m9 = 2,
+	)
+	crate_name = "crate"
+	
+/datum/supply_pack/security/armory/wt550ammo
+	name = "Recalled Ammo Pack"
+	desc = "Contains four 20-round magazine for the Recalled WT-550 Auto Rifle. \
```

Comment:
looks like it contains two. does each box have two?

Reply:
Must have accidentally deleted one while debugging it

Reply:
added the amount back to normal